===================================================================
RCS file: /cvs/liboil/liboil/liboil/powerpc/conv.c,v
rcsdiff: /cvs/liboil/liboil/liboil/powerpc/conv.c,v: warning: Unknown phrases like `commitid ...;' are present.
retrieving revision 1.2
retrieving revision 1.3
diff -u -r1.2 -r1.3
--- conv.c	2005/12/04 18:51:55	1.2
+++ conv.c	2006/02/03 18:06:37	1.3
@@ -33,8 +33,8 @@
 
 
 #define DEFINE_CLIPCONVERT_POWERPC(dsttype,srctype,minval,maxval,insn1,insn2)					\
-void clipconv_##dsttype##_##srctype##_powerpc(type_##dsttype *dst,	\
-	int dst_stride, type_##srctype *src, int src_stride, int n)		\
+void clipconv_##dsttype##_##srctype##_powerpc(oil_type_##dsttype *dst,	\
+	int dst_stride, oil_type_##srctype *src, int src_stride, int n)		\
 {									\
 	double min = (minval);						\
 	double max = (maxval);						\
